sympy/integrals/transforms.py

```diff
@@ -1151,7 +1709,14 @@ class LaplaceTransform(IntegralTransform):
     _name = 'Laplace'
 
     def _compute_transform(self, f, t, s, **hints):
-        return _laplace_transform(f, t, s, **hints)
+        try:
+            hints['doit'] = True
+            return _laplace_apply_rules(f, t, s, **hints)
```

Comment:
I don't understand what the `doit` flag does here. Is that so that the `_laplace_apply_rules` function could be used in a different context?

Is `doit` already a flag used by `laplace_transform`?

Maybe `laplace_apply_rules` could use a different parameter name for this to avoid conflicting with whatever `doit` does.

Reply:
Yes, that flag makes just that possible. This is how I understood your original suggestion to write the rules engine such that it can be used, e.g., by integrate.

The doit flag does not exist yet in the Laplace transform.

Reply:
Okay so if this flag only exists for the purpose of using `_laplace_apply_rules` then it can have a more descriptive name. To me `doit` in this context suggests "evaluate any integrals". How can we describe in a name what this flag is for?

Also we don't need `hints['doit'] = True`. It's just:
```python
return _laplace_apply_rules(f, t, s, doit=True, **hints)
```

Reply:
Thanks, I will do precisely this. I used the other strategy by learning from the present (and apparently very old) code.